test/topology_custom/test_replace.py

```diff
@@ -73,12 +76,50 @@ async def test_replace_different_ip_using_host_id(manager: ManagerClient) -> Non
     await wait_for_token_ring_and_group0_consistency(manager, time.time() + 30)
 
 @pytest.mark.asyncio
-async def test_replace_reuse_ip(manager: ManagerClient) -> None:
+async def test_replace_reuse_ip(request, manager: ManagerClient) -> None:
```

Comment:
Please start 2 jenkins jobs with https://jenkins.scylladb.com/job/scylla-master/job/byo/job/byo_build_tests_dtest/ 
- one in x86 mode (default)
- one in aarch64 mode (can be set with `ARCH` parameter at the bottom of build page)

to run this new test 100 times in all modes

Reply:
ok

Reply:
See https://jenkins.scylladb.com/job/scylla-master/job/byo/job/byo_build_tests_dtest/2174/parameters/ for example of one recent build by @margdoc to test https://github.com/scylladb/scylladb/pull/17960

Reply:
Better example: https://jenkins.scylladb.com/job/scylla-master/job/byo/job/byo_build_tests_dtest/2169/parameters/

this was running a single test.py test on aarch64 200 times

Reply:
https://jenkins.scylladb.com/job/scylla-master/job/byo/job/byo_build_tests_dtest/2175/

Reply:
>  https://jenkins.scylladb.com/job/scylla-master/job/byo/job/byo_build_tests_dtest/2175/

You're running `test_topology_upgrade` here (`INCLUDE_TESTS` parameter)
and on `cdc_upgrade_test_v2` branch of `git@github.com:scylladb/scylla-dev.git` repo. (`SCYLLA_REPO`, `SCYLLA_BRANCH` params)

Reply:
BTW I think this job can only use a subset of predefined repos, one of them being `git@github.com:scylladb/scylla-dev.git`, you should be able to push a branch to this repo and then use that branch.

Reply:
Have no idea what happened to it, I remember I set the proper parameters. Reran https://jenkins.scylladb.com/job/scylla-master/job/byo/job/byo_build_tests_dtest/2176.

Reply:
> Have no idea what happened to it, I remember I set the proper parameters. Reran https://jenkins.scylladb.com/job/scylla-master/job/byo/job/byo_build_tests_dtest/2176.

Crashed with some meaningless exception. Has `SCYLLA_FORK_REPO` stopped working for this job? Why? I used to use this job in this way. @yaronkaikov 

https://jenkins.scylladb.com/job/scylla-master/job/byo/job/byo_build_tests_dtest/2177 - pushed to `scylla-dev.git`.

Reply:
only `test_replace.test_replace_different_ip` failed, which I didn't touch in this PR